packages/core/src/external/carequality/pd/validating-pd.ts

```diff
@@ -0,0 +1,189 @@
+import {
+  Address as FHIRAddress,
+  ContactPoint,
+  Identifier,
+  Patient as FHIRPatient,
+} from "@medplum/fhirtypes";
+import { getStateEnum, USState } from "../../../domain/geographic-locations";
+import { Address } from "../../../domain/address";
+import { Contact } from "../../../domain/contact";
+import { PatientData, PersonalIdentifier } from "../../../domain/patient";
+import { isContactType } from "../../fhir/patient/index";
+import { MetriportError } from "../../../util/error/metriport-error";
+import status from "http-status";
+
+export class IHEGatewayError extends MetriportError {
+  constructor(
+    message: string,
+    public iheErrorCode: string,
```

Comment:
Missing the `cause` parameter. I suggest we keep the same pattern as `MetriportError`, `cause` being the second parameter.